src/events/onceReady.js

```diff
@@ -3,42 +3,39 @@ const { Team } = require('discord.js');
 let retries = 0;
 
 module.exports = class extends Event {
-	constructor(...args) {
-		super(...args, {
+	constructor(context) {
+		super(context, {
 			once: true,
 			event: 'ready'
 		});
 	}
 
 	async run() {
+		const { client } = this.context;
 		try {
-			await this.client.fetchApplication();
+			await this.context.client.fetchApplication();
 		} catch (err) {
 			if (++retries === 3) return process.exit();
-			this.client.emit('warning', `Unable to fetchApplication at this time, waiting 5 seconds and retrying. Retries left: ${retries - 3}`);
+			client.emit('warning', `Unable to fetchApplication at this time, waiting 5 seconds and retrying. Retries left: ${retries - 3}`);
```

Comment:
isn't that subtraction reversed?

Reply:
wdym?

Reply:
retries - 3 would yield -2, -1, then 0

Reply:
That's Klasa for you, I didn't write that LOL